vital/tests/test_local_geo_cs.cxx

```diff
@@ -50,7 +50,7 @@ TEST_F(local_geo_cs, read_local_geo_cs_from_file)
   valid = read_local_geo_cs_from_file(lgcs, geo_origin_valid);
   EXPECT_TRUE(valid);
   // Expect the local geo cs to be valid
-  kv::vector_3d origin = lgcs.origin().location(crs);
+  kv::vector_3d origin = lgcs.origin().location();
```

Comment:
This change only works because the default `crs` happens to be `kv::SRID::lat_lon_WGS84`